<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1287</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VI4310_Demo(v0.1)</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/icon_1.png</normaloff>images/icon_1.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>3D点云：</string>
      </property>
      <layout class="QGridLayout" name="gridLayout" rowstretch="15,1">
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <widget class="GLWidget" name="widget" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,0,1,1,1,1,1,1,5">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="tof_peak_change_toolButton">
           <property name="text">
            <string>tof/peak</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>增益：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="gain_lineEdit">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="front_toolButton">
           <property name="text">
            <string>front</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>images/ccViewYpos.png</normaloff>images/ccViewYpos.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="side_toolButton">
           <property name="text">
            <string>side</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>images/ccViewXneg.png</normaloff>images/ccViewXneg.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="down_toolButton">
           <property name="text">
            <string>down</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>images/ccViewZpos.png</normaloff>images/ccViewZpos.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="filter_radioButton">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1287</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="openFile_action"/>
    <addaction name="saveFile_action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openFile_action"/>
   <addaction name="saveFile_action"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="left_dockWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>深度图(tof)：</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2" rowstretch="9,0">
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="tof_Histogram_pushButton">
            <property name="text">
             <string>直方图</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>images/ccHistogram.png</normaloff>images/ccHistogram.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="myLabel" name="tof_label">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>强度图(peak)：</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3" rowstretch="9,1">
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="myLabel" name="peak_label">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="peak_Histogram_pushButton">
            <property name="text">
             <string>直方图</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>images/ccHistogram.png</normaloff>images/ccHistogram.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QToolBox" name="config_toolBox">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>300</width>
          <height>433</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Page 1</string>
        </attribute>
        <widget class="QPushButton" name="play_pushButton">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>120</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>play</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>300</width>
          <height>433</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Page 2</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowTitle">
    <string>信息监控：</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="Control_textEdit" name="control_log_textEdit">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="openFile_action">
   <property name="icon">
    <iconset resource="imgsource.qrc">
     <normaloff>:/new/prefix1/images/ccOpen.png</normaloff>:/new/prefix1/images/ccOpen.png</iconset>
   </property>
   <property name="text">
    <string>打开本地文件</string>
   </property>
  </action>
  <action name="saveFile_action">
   <property name="icon">
    <iconset>
     <normaloff>images/ccSave.png</normaloff>images/ccSave.png</iconset>
   </property>
   <property name="text">
    <string>保存文件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>myLabel</class>
   <extends>QLabel</extends>
   <header>mylabel.h</header>
  </customwidget>
  <customwidget>
   <class>Control_textEdit</class>
   <extends>QTextEdit</extends>
   <header>control_textedit.h</header>
  </customwidget>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="imgsource.qrc"/>
 </resources>
 <connections/>
</ui>
